src/systems/reset_stat_effectors.rs

```diff
@@ -0,0 +1,10 @@
+use crate::*;
+
+pub fn reset_stat_effectors_system(stats: &mut Components<StatSet<Stats>>) -> SystemResult {
+         for stat in stats.iter_mut() {
+            for mut s in stat.stats.values_mut() {
+                s.value_with_effectors = s.value;
```

Comment:
Isn't this already managed by this? https://github.com/jojolepro/minigene/blob/master/src/systems/apply_effector.rs#L13

Reply:
The problem with this is that `value_with_effectors` is set with `0` by default and remains `0` forever. This occurs because it's only being reset (`value_with_effectors = value`  ) when it is affected by an `effector` (meeting this join `for (stat, effector) in join!(&mut stats && &effectors)`. So I create this system that reset every stat, even if the stats are not affected by an effector. With this system, I assure that the stat `value_with_effectors` will be equal to `value` every init of the frame.

Reply:
Maybe this should be done by `apply_effectors_system` or change the default value of `value_with_effectors` to meet `value`

Reply:
Actually `value_with_effectors = value` happens no matter if you have an effector or not. What matters is that you have an `EffectorSet` component.
I agree with you that value_with_effectors should be initialised with the `value` instead of 0, let's do that!

Reply:
>  I agree with you that value_with_effectors should be initialised with the value instead of 0, let's do that!
I did the change in game_features. A `cargo update` will fix it for you :)

Reply:
Great! You do the work on `minigene` or you want me to do a PR?

Reply:
hmmm I think that if you just remove the `reset_stat_effectors_system` it should work without any other changes?

Reply:
I tested here and removing this system all opponents don't move anymore x_x